pkg/apis/apps/v1/conversion.go

```diff
@@ -115,47 +73,29 @@ func Convert_apps_Deployment_To_v1_Deployment(in *apps.Deployment, out *appsv1.D
 		delete(out.Annotations, appsv1.DeprecatedRollbackTo)
 	}
 
-	if err := Convert_apps_DeploymentStatus_To_v1_DeploymentStatus(&in.Status, &out.Status, s); err != nil {
-		return err
-	}
 	return nil
 }
 
 func Convert_apps_DaemonSet_To_v1_DaemonSet(in *apps.DaemonSet, out *appsv1.DaemonSet, s conversion.Scope) error {
+	if err := autoConvert_apps_DaemonSet_To_v1_DaemonSet(in, out, s); err != nil {
+		return err
+	}
+
 	out.ObjectMeta = in.ObjectMeta
 	out.Annotations = deepCopyStringMap(out.Annotations) // deep copy annotations because we change them below
-
 	out.Annotations[appsv1.DeprecatedTemplateGeneration] = strconv.FormatInt(in.Spec.TemplateGeneration, 10)
-	if err := Convert_apps_DaemonSetSpec_To_v1_DaemonSetSpec(&in.Spec, &out.Spec, s); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.Status, &out.Status, 0); err != nil {
-		return err
-	}
 	return nil
 }
 
 func Convert_apps_DaemonSetSpec_To_v1_DaemonSetSpec(in *apps.DaemonSetSpec, out *appsv1.DaemonSetSpec, s conversion.Scope) error {
```

Comment:
can delete Convert_apps_DaemonSetSpec_To_v1_DaemonSetSpec if we just delegate to the autogenerated method

Reply:
I did that initially, but unfortunately I can't - code-generator doesn't generate the public method for it, because auto-generated ones contains a warning.

Reply:
done